### PR TITLE
Adding `SubjectOnDiskTrialPass::computeKinematicValues`

### PR DESCRIPTION
**Pull request by nickbianco**
Fixes https://github.com/keenon/AddBiomechanics/issues/227.

This PR accomplishes two things:

-  Adds `SubjectOnDiskTrialPass::computeKinematicValues` for computing all kinematic values in a trial while setting all dynamics values to zero.
-  `SubjectOnDiskTrialPass::computeValuesFromForcePlates` calls `SubjectOnDiskTrialPass::computeKinematicValues` if incorrect foot body name is provided.

### CODE REVIEW

File: dart/biomechanics/SubjectOnDisk.cpp [hunk 1]
```diff
@@ -2147,12 +2148,29 @@ void SubjectOnDiskTrialPass::computeValuesFromForcePlates(
...
+
     footBodies.push_back(footBody);
     footIndices.push_back(footBody->getIndexInSkeleton());
   }
+
+  if (foundNonexistentFootBody) {
+    std::cout << "WARNING: One of the foot bodies specified does not exist in "
+                 "the skeleton. Skipping dynamics calculations..."
+              << std::endl;
+    computeKinematicValues(skel, timestep, poses, rootHistoryLen,
```
Comment (keenon) [inline dart/biomechanics/SubjectOnDisk.cpp:2169]: Should we maybe move this line prior to the if statement, so it always gets called, and then just exit early in here? That way we can delete the code down below that computes the same values, and avoid duplication.

**Review by keenon — COMMENTED** [review]
Minor DRY thing

File: dart/biomechanics/SubjectOnDisk.cpp (hunk 1 above)
Comment (nickbianco) [reply dart/biomechanics/SubjectOnDisk.cpp:2169]: Good idea, will do!

Comment (nickbianco) [reply dart/biomechanics/SubjectOnDisk.cpp]: Done.

**Comment by nickbianco** [comment]
@keenon finished refactoring `computeKinematicValues` and `computeValuesFromForcePlates`. Let me know what you think.

**Review by keenon — APPROVED** [review]
LGTM!

**Pull request merged**